contracts/Token.sol

```diff
@@ -228,10 +283,26 @@ contract Token is Initializable, IToken, IERC20 {
     _;
   }
 
-  modifier isActor(address c) {
+  modifier isOwnerActor(address c) {
```

Comment:
@afmfe-iul I decided to remove the plain `isActor` modifier in favour of more verbose ones (so you can know which one of your parameter is actually not an actor).